src/scss/login.module.scss

```diff
@@ -98,32 +103,37 @@ $white: #fff;
   font-size: 1rem;
   transform: rotate(-90deg) translateY(50%);
   float: left;
-  margin-left: -4rem;
-  position: relative;
-  top: 45%;
+  margin-top: 10rem;
+  margin-left: -8rem;
   letter-spacing: 0.2rem;
 }
+@media only screen and (max-width: 561px) {
+  .coords {
+    font-size: 0.6rem;
+  }
+}
 
 .ellipses-container {
-  width: 35rem;
-  height: 35rem;
+  width: 20vw;
+  height: 20vw;
   border-radius: 50%;
   margin: 0 auto;
   position: relative;
   top: 1.5rem;
+  background: url(https://i.giphy.com/media/l0iVAuEtPF5KRbqmvd/source.gif);
+  background-size: cover;
+  opacity: 0.5;
+}
 
-  .greeting {
-    position: absolute;
-    top: 2rem;
-    left: 6.4rem;
-    margin: 0 auto;
-    opacity: 0.5;
+@media only screen and (max-width: 600px) {
+  .ellipses-container {
+    width: 30vw;
+    height: 30vw;
   }
 }
 
 .login-github {
   text-align: center;
-  margin: 0 auto;
   text-transform: uppercase;
   margin-left: 6rem;
   margin-top: 6rem;
```

Comment:
```suggestion
  margin-top: 9rem;
```

Reply:
Adding this and other size increases would enable scrolling again on a login page, not sure if that would be a great UX choice or not.

Reply:
It wasn't added scrolling to any devices when checked locally. Could you send me examples if you can reproduce the issue?

Reply:
Sure, checkout the scroll bars.

![image](https://user-images.githubusercontent.com/48270786/88451538-06467e00-ce75-11ea-8933-94a2e283d7de.png)

![image](https://user-images.githubusercontent.com/48270786/88451549-1c543e80-ce75-11ea-9e26-511d8374a31d.png)

Reply:
Okay, thanks for the heads up here. For solving this:

could you try removing the `margin-top` for the github-login and place it in the page using `margin-bottom` and setting `overflow-Y: false` for the parent div holding it. 

This way changing the scale would not result in scrollbars and simultaneously solve the button `center` alignment too

Reply:
`overflow-Y` doesn't have a property false, did you mean hidden? Wow I sound like google on wrong results lol

Reply:
I tried and couldn't get anything to fix the button layout. I'm actually very curious why `margin-top:9rem` is the magic number where the div positions itself vertically and horizontally.

Reply:
Cool, for now we can do with the 9rem hack ig